### ISSUE TITLE
Кнопки VPN в приложении.

### ISSUE THREAD

**Issue by Vitalik93** [AdguardTeam/AdguardForiOS#1742]
Хотелось бы узнать, зачем в релиз 4.1.1 были добавлены кнопки VPN? Если пользователь, как я, никогда не собирается устанавливать это приложение, зачем тогда они нужны? Расцениваю их введение как наличие встроенной рекламе, что тем более странно видеть в Premium версии, за которую были заплачены деньги и от фирмы, которая считает себя борцом с этой самой рекламой.

**Comment by zzebrum** [comment]
обязательно будет добавлена такая возможность 👍

**Comment by vozersky** [comment]
в 4.2  будет

**Comment by ameshkov** [comment]
Here's what I propose:

1. Long tap on ANY icons (actually, on the container where these icons are located) - "What icons would you like to see here"?
2. Show a standard system multi-select dialog with the following options: 
    * Safari Protection
    * DNS Protection
    * AdGuard VPN

Here's an example of the system multi-select dialog: http://htmlbook.ru/html/select/multiple (open in mobile Safari to see what I mean).

**Comment by iDjay** [comment]
I don‘t speak russian but we need a „split tunnel without VPN icon“ option.

The VPN icon shows at a glance whether I'm connected to AdGuard VPN or not. The prerequisite for this is that AdGuard DNS runs in the Split Tunnel, which currently only works with VPN symbol.

**Comment by ameshkov** [comment]
@iDjay this task is about icon buttons on the main screen, the ones that are right above "Protection is enabled/disabled" label :)

**Comment by zznosar** [comment]
I hope this is still scheduled for 4.2. It will be great to configure those icons. Especially for people who don’t use the adGuard VPN.

**Comment by Shadowspear123** [comment]
Yes please add the option to  remove the third vpn icon on the home screen as it is annoying to those who don’t plan on using it.

**Comment by zeezeepiggy** [comment]
Bump. @ameshkov, is this feature still on track for version 4.3? I rarely use AdGuard VPN on my iphone. Only on my mac and ipad. so having configurable icons to hide the adGuard VPN option will be a great addition

**Comment by ameshkov** [comment]
Definitely not v4.3 since it's all about web extensions and we don't want to delay it by other tasks.

**Comment by zeezeepiggy** [comment]
@ameshkov . Understood. Thanks. I’m very much excited for the Safari web extension. Do we have an ETA?

**Comment by ameshkov** [comment]
@zeezeepiggy next week most likely
